### PR TITLE
Added Ombi

### PR DESCRIPTION
**Pull request by tidusjar**
Added my project Ombi !

### CODE REVIEW

**Comment by shiftkey** [comment]
@tidusjar the file needs to have a `.yml` extension so that it gets parsed as site data

**Comment by tidusjar** [comment]
Hey! Sorry about that!

Just fixed that.

File: _data/projects/ombi.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  Want a Movie or TV Show on Plex or Emby? Use Ombi! 
+  Ombi is a user friendly interface for family and friends to request new content on your media server.
+site: https://github.com/tidusjar/ombi
+tags:
+- ".NET"
+- "TypeScript"
+- "Angular"
+upforgrabs:
+  name: help wanted
+  link: https://github.com/tidusjar/ombi/labels/help wanted
```
Comment (shiftkey) [inline _data/projects/ombi.yml]: I _think_ this needs to be URL-encoded to address the current CI failure: `https://github.com/tidusjar/ombi/labels/help%20wanted`

**Comment by shiftkey** [comment]
@tidusjar thanks!

**Pull request merged**